code/modules/cooking/machinery/cooking_machines/oven.dm

```diff
@@ -51,12 +51,13 @@
 
 /obj/machinery/appliance/cooker/oven/update_icon()
 	if (!open)
-		if (!stat)
-			icon_state = "ovenclosed_on"
-		else
-			icon_state = "ovenclosed_off"
+		icon_state = "ovenclosed"
 	else
 		icon_state = "ovenopen"
+	cut_overlays()
+	if (!stat)
+		var/glow = image('icons/obj/cooking_machines.dmi', "oven_on", EFFECTS_ABOVE_LIGHTING_LAYER)
+		add_overlay(glow)
```

Comment:
```suggestion
		add_screen_overlay(glow)
```

Reply:
It looks like `add_screen_overlay()` is only defined on `/obj/machinery/vending/`.

Reply:
`make_screen_overlay()` also doesn't accept an `image()`
Testing it, there's too much bloom, and it causes the heating element glow to layer over cookware.